sparse2d/python/cmake/Modules/BuildSparse2D.cmake

```diff
@@ -11,14 +11,13 @@ ExternalProject_Add(sparse2d
     # GIT_TAG master
     DEPENDS cfitsio
     CONFIGURE_COMMAND cmake ../sparse2d
-        -DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}/extern
+        -DCMAKE_INSTALL_PREFIX=${SPARSE2D_INSTALL_DIR}/..
```

Comment:
You might want to check this. The builds failed on Travis and and also failed locally for me.

Reply:
On it, I expected some issue, but I added it assuming it is still a good fix to get in